CLAUDE.md

```
@@ -0,0 +1,326 @@
+export CLOUDKIT_KEY_ID="your-key-id"
+export CLOUDKIT_KEY_FILE="$HOME/.cloudkit/bushel-private-key.pem"
+export CLOUDKIT_CONTAINER_ID="iCloud.com.yourcompany.Bushel"  # Optional, has default
+```
+
+## Architecture
+
+### Core Data Flow
+
+The application follows a pipeline architecture:
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)